src/context/Theme.tsx

```diff
@@ -155,32 +155,24 @@ export const outlineSx: SxProps<Theme> = {
     border: `1px solid ${slate[200]}`,
 };
 
-export const darkGlassBkgColor = 'rgba(172, 199, 220, 0.12)';
+export const darkGlassBkgColor = 'rgb(172, 199, 220, 0.12)';
```

Comment:
`rgb` is a three-channel format, so the fourth parameter should be removed.

Reply:
Thanks.... now I cannot remember is this is because I was trying to remove the the Opacity or add it back in :(